docker_images/diffusers/requirements.txt

```diff
@@ -2,11 +2,18 @@ starlette==0.25.0
 api-inference-community==0.0.30
 huggingface_hub==0.14.1
 safetensors==0.3.1
-diffusers==0.16.1
+# diffusers==0.16.1
+git+https://github.com/huggingface/diffusers@main
```

Comment:
Any reason to use main ? ( It works, but it's harder to know when something was built, leading to confusion when some features is missing from here, while it is actually on main)

Reply:
you're right!, @patrickvonplaten what would be a good commit to pin here? I can update when 0.17 come out

Reply:
Commit is the best in between !